microverstrail.css

```diff
@@ -0,0 +1,13 @@
+#navbar {
+    position: fixed;
+    top: 0;
+  }
+  
+  #welcome-section {
+    height: 100vh
+  }
+  @media (max-width: 768px) {
+    body{
+      background-color: brown;
+    }
+  }
```

Comment:
Please, kindly consider adding an empty line after you finish your code :smile: :smile: :smile:

Reply:
Good 👍 job